zeppelin-zengine/src/test/java/org/apache/zeppelin/notebook/NotebookTest.java

```diff
@@ -210,6 +211,21 @@ public void testPersist() throws IOException, SchedulerException, RepositoryExce
   }
 
   @Test
+  public void testCreateNoteWithSubject() throws IOException, SchedulerException, RepositoryException {
+    AuthenticationInfo subject = new AuthenticationInfo("user1");
+    Note note = notebook.createNote(subject);
+
+    Notebook notebook = new Notebook(
```

Comment:
yeah that's true, but the question is why you instantiate `notebook` with `new Notebook` on line 218 if you already used different instance of it to create `note` on line 216.  you can reuse same instance and just delete instantiation at line 218 , isn't it?